src/python/pants/reporting/html_reporter.py

```diff
@@ -317,7 +317,7 @@ def handle_output(self, workunit, label, s):
         output_files[path] = f
       else:
         f = output_files[path]
-      f.write(self._htmlify_text(s).encode('utf-8'))
+      f.write(self._htmlify_text(s))
```

Comment:
This reporter code is worth a sanity check. It seemed to be expecting unicode in most places, as Py3 yells about all the original uses of bytes. 

Using unicode works when testing locally for me, semi-tested because when this code was failing the output of Pants was unformatted until fixing this.

Reply:
If you're able to actually invoke pants with python 3, you can confirm that clicking around in `./pants server --open` looks reasonable.

Reply:
But yea, HTML shouldn't contain any binary data, so this seems reasonable.

Reply:
I won't be able to test that yet, unfortunately, because `./pants` does not yet fully compile with Python 3 virtual env. 

Fortunately, the changes we're making with all of these PRs are innocuous for now—so long as they don't break Python 2—because we haven't yet turned on Python 3 for anything. So, if we realize in a few days this change was a mistake, we can easily revert back.